docs/assets/dev/js/editor/document/component.md

```diff
@@ -0,0 +1,62 @@
+- ![#43f7ff](https://via.placeholder.com/15/43f7ff/000000?text=+)  `Widget ID` &nbsp;&nbsp;- `5412776`
+
+  ![legend-section-widget-tabs](_images/legend-section-widget-tabs.png)
+
+## Component -- `$e.components.get('document')`
+
+* **Name**: Document.
+* **Description**: `Document` is used as namespace of describing changes in the document made by the command.
+
+## All **Document** subcomponents
+| Component Name                 | Access Component                         | Description
+|--------------------------------|------------------------------------------|-----------------------
+| [Dynamic](dynamic/readme.md)   | `$e.components.get('document/dynamic')`  | Dynamic enable, disable and change.
+| [Elements](elements/readme.md) | `$e.components.get('document/elements')` | Elements manipulation (create, edit, remove, etc.).
+| [History](history/readme.md)   | `$e.components.get('document/history')`  | Provides a way to record/manipulate the commands history.
+| [Repeater](repeater/readme.md) | `$e.components.get('document/repeater')` | Provides a way to communicate with repeaters.
+| [Save](save/readme.md)         | `$e.components.get('document/save')`     | Responsible for saving the document (either publish/draft).
+| [UI](ui/readme.md)             | `$e.components.get('document/ui')`       | Irregular component for handling UI shortcuts.
+
+## Component `document` -- Utils
+| Util Name                                                                 | Access Util                                               
+|---------------------------------------------------------------------------|-----------------------------------------------------------|
+| [findViewRecursive](#util----ecomponentsgetdocumentutilsfindviewrecursive)| `$e.components.get('document').utils.findViewRecursive()` |
+| [findViewById](#util----ecomponentsgetdocumentutilsfindviewbyid)          | `$e.components.get('document').utils.findViewById()`      |
+| [findContainerById](#util----ecomponentsgetdocumentutilsfindcontainerbyid)| `$e.components.get('document').utils.findContainerById()` |
+
+## _Util_ -- `$e.components.get('document').utils.findViewRecursive()`
+* **Name**: findViewRecursive.
+* **Description**: Find view recursively.
+* **Returns**: `{Array}`
+
+   | Property     | Type                  | Requirement   | Description |
+   |---           |---                    |---            |---          |
+   | _parent_     | {Marionette.View}     | _required_    | Parent view.
+   | _key_        | {String}              | _required_    | Key of the view used to  find the view. e.g: id.
+   | _value_      | {String}              | _required_    | Value of the view used to find the view. e.g: `98c3441`.
+   | _multiple_   | `{Boolean}`           | **optional**  | default: `{false}`. allow multiple results.
+
+## _Util_ -- `$e.components.get('document').utils.findViewById()`
+* **Name**: findViewById.
+* **Description**: Find view by id.
```

Comment:
The name and description don't explain anything more than the title. please remove it here and below